app/views/layouts/_empty.html.haml

```diff
@@ -0,0 +1,19 @@
+.blank-slate-pf
+  .blank-slate-pf-icon
+    %span.pficon.pficon-add-circle-o
+  %h1= _("Add a Provider")
+  %p
+    = _("You don't have any providers to display. Please add a Provider.")
+  %p
+    = _("Learn more about this")
+    %a{:href => 'https://manageiq.org/docs/get-started/add-a-provider',
+       :target => '_blank'}
+      = _("in the documentation.")
+  .blank-slate-pf-main-action
+    = link_to(_("Add a Provider"),
+      {:action => 'new'},
+      :class => 'btn btn-primary btn-lg',
+      "data-miq_sparkle_on" => true,
+      "data-miq_sparkle_off" => true,
+      "data-method" => :post,
+      :title => _("Add a new %{provider_type} Provider") % {:provider_type => type})
```

Comment:
We need to avoid doing string arithmetics with gettext strings as much as possible (i.e. we should avoid string concatenations). In this case, may I suggest to add the full string from parent view:

```ruby
= render :partial => 'layouts/empty', :locals => {:add_text => _("Add a new Network Provider")}
```

and then just render the text here?

Reply:
@mzazrivec updated.